src/styles/main.scss

```diff
@@ -0,0 +1,126 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.container {
+  width: 100%;
+  min-height: 100vh;
+  padding: 10px;
+}
+
+h2 {
+  font-family: cursive;
+  text-decoration: underline;
+}
+
+.todo-app {
+  width: 100%;
+  max-width: 540px;
+  background-color: white;
+  box-shadow: 0 1px 10px rgba(0, 0, 0, 0.11);
+  margin: 100px auto;
+  padding: 40px 30px 70px;
+  border-radius: 20px;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 25px 0;
+  box-shadow: 0 1px 10px rgba(0, 0, 0, 0.062);
+  border-radius: 10px;
+  padding: 10px 10px;
+  width: 100%;
+}
+
+input {
+  flex: 1;
+  border: none;
+  outline: none;
+  background-color: transparent;
+  padding: 10px;
+}
+
+button {
+  border: none;
+  border: 1px solid rgba(0, 0, 0, 0.129);
+  outline: none;
+  padding: 15px 40px;
+  background-color: rgba(63, 70, 63, 0.511);
+  color: white;
+  font-size: 16px;
+  cursor: pointer;
+  border-radius: 30px;
+}
```

Comment:
- [ OPTIONAL] I suggest you to apply the wireframe. Your button looks so big and overflowing the next line. You can use a trash icon and add click event on the text to edit it. Also, you added checkboxes but you check it via button. It wouldn't be a good UX design. 
![image](https://github.com/MustafaSamiShirzad/to-do-list/assets/91486791/05ceaaaf-f16f-40fa-9667-49e77da5a782)